client/components/Grid.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+// eslint-disable-next-line import/extensions
+import CardsList from './CardsList.jsx';
+// eslint-disable-next-line import/extensions
+import GridHeader from './GridHeader.jsx';
+
+const Container = styled.div`
+  background-color: rgb(250,250,250);
+  width: 100%;
+  max-width: 1440px;
+  max-height: 838px;
+  margin-top: 48px;
+  color: #101820;
+  font-family: Roboto, "Helvetica Neue", sans-serif;
+`;
+
+class Grid extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // eslint-disable-next-line no-undef
+      currentId: window.location.pathname.split('/')[1],
```

Comment:
I'm not sure about this, but what would happen if you handled this by changing state in componentDidMount before calling this.getInfo? You could 'possibly' get away without having to disable the linter for this line. Just something to play around with, maybe.

Reply:
The linter says that window is undefined because it doesn't like global variables. I'm not sure I understand what you mean by 'handled this by changing state in componentDidMount before calling this.getInfo' 🤔